packages/lodestar/src/network/peers/peerManager.ts

```diff
@@ -492,6 +492,7 @@ export class PeerManager {
 
     this.logger.verbose("peer disconnected", {peer: prettyPrintPeerId(peer), direction, status});
     this.networkEventBus.emit(NetworkEvent.peerDisconnected, peer);
+    this.reqResp.prune(peer);
```

Comment:
If prune() here only prunes rate limit data make the name longer to clarify that it does exactly that and nothing else